packages/vuetify/src/components/VBadge/VBadge.sass

```diff
@@ -55,6 +55,9 @@
 
   .v-icon
     color: inherit
+    font-size: $badge-font-size
+    margin: $badge-icon-margin
```

Comment:
This value only worked in v2 because badges had 2px extra inline padding and weren't display flex, now it makes icons way too cramped if you also have text next to them. 

@nekosaur https://github.com/vuetifyjs/vuetify/commit/87cc5af17091bfa4604ae7613479de0772407e0f / https://github.com/vuetifyjs/vuetify/commit/8cd727ebe343821d5475d9bbb03176866ed505b1

Reply:
That means the old values for display and padding need to be brought back too, unless @nekosaur had some specific reason otherwise.

Reply:
No I don't think so, or I don't remember.